packages/@idemsInternational/gdrive-tools/src/commands/authorize.ts

```diff
@@ -1,6 +1,10 @@
 import fs from "fs";
-import readline from "readline";
+import chalk from "chalk";
 import { driveactivity_v2, drive_v3, google } from "googleapis";
+import http from "http";
+import opn from "open";
+import url from "url";
+import destroyer from "server-destroy";
```

Comment:
This package doesn't seem to be maintained and is only 16 lines of code anyway. Better to just include manual methods to destroy server
https://github.com/isaacs/server-destroy/blob/master/index.js